docs/v3/get-started/quickstart.mdx

```diff
@@ -1,179 +1,125 @@
+    ```bash
+    git clone https://github.com/PrefectHQ/quickstart && cd quickstart
+    uv run 01_getting_started.py
+    ```
+    If all went well, you'll see a link to see its execution graph in the Prefect UI followed by a flurry of state changes and logs as Prefect dynamically discovers and executes your workflow.
+
+    ```bash {11} [expandable]
+    00:30:53.633 | INFO    | Flow run 'airborne-ringtail' - Beginning flow run 'airborne-ringtail' for flow 'main'
+    00:30:53.638 | INFO    | Flow run 'airborne-ringtail' - View at https://app.prefect.cloud/account/...
+    00:30:53.685 | INFO    | Task run 'get_customer_ids-136' - Finished in state Completed()
+    00:30:54.512 | INFO    | Task run 'process_customer-d9b' - Finished in state Completed()
+    00:30:54.518 | INFO    | Task run 'process_customer-113' - Finished in state Completed()
+    00:30:54.519 | INFO    | Task run 'process_customer-1c6' - Finished in state Completed()
+    00:30:54.519 | INFO    | Task run 'process_customer-30d' - Finished in state Completed()
+    00:30:54.520 | INFO    | Task run 'process_customer-eaa' - Finished in state Completed()
+    00:30:54.523 | INFO    | Task run 'process_customer-b2b' - Finished in state Completed()
+    00:30:54.523 | INFO    | Task run 'process_customer-90a' - Finished in state Completed()
+    ...Ommitted for brevity in this tutorial
+    00:30:54.553 | INFO    | Task run 'process_customer-4af' - Finished in state Completed()
+    00:30:54.555 | INFO    | Task run 'process_customer-e66' - Finished in state Completed()
+    00:30:54.556 | INFO    | Task run 'process_customer-e7e' - Finished in state Completed()
+    00:30:54.556 | INFO    | Task run 'process_customer-5bd' - Finished in state Completed()
+    00:30:54.557 | INFO    | Task run 'process_customer-05d' - Finished in state Completed()
+    00:30:54.558 | INFO    | Task run 'process_customer-b60' - Finished in state Completed()
+    00:30:54.701 | INFO    | Flow run 'airborne-ringtail' - Finished in state Completed('All states completed.')
+    ```
+    </Step>
+    <Step title="Where should it run?">
+    At this point, you've successfully run a Prefect `flow` locally. Let's get it running off of your machine! Again, for simplicity, 
+    we'll deploy this workflow to Prefect's Serverless Compute (so we don't have to wrangle any infrastructure). We'll tell Prefect to clone
+    `https://github.com/PrefectHQ/quickstart` and invoke `01_getting_started.py:main`.
+    ```python
+    uvx prefect-cloud deploy 01_getting_started.py:main \
+    --from https://github.com/PrefectHQ/quickstart
+    ```
+    If all went well, you've created your first deployed workflow. 
+    
+    Since we know what code to run and where to run it, this can now be run remotely. 
+    To do that, run the following command and visit the returned link to see the run's status in the Prefect UI:
+    ```python 
+    uvx prefect-cloud run main/main
```

Comment:
Makes me wonder if we should specify a deployment name with `--name` in the above command or change the default to add a suffix or something